src/background/utils/preinject.js

```diff
@@ -153,7 +154,6 @@ function prepareScript(script, index, scripts) {
     // Firefox lists .user.js among our own content scripts so a space at start will group them
     `\n//# sourceURL=${extensionRoot}${ua.isFirefox ? '%20' : ''}${name}.user.js#${id}`,
   ].join('');
-  cache.put(dataKey, injectedCode, TIME_KEEP_DATA);
```

Comment:
Removing this line as the cache seems to be unused.

Reply:
It's used by InjectionFeedback.

Reply:
Oh, my bad...